package.json

```diff
@@ -19,7 +19,7 @@
     "generate:sdk": "pnpm run -r build:codegen-sdk && pnpm run -r generate:sdk",
     "generate:test": "pnpm run -r build:codegen-test && pnpm run -r generate:test",
     "generate": "run-s generate:clean generate:doc generate:sdk",
-    "lint": "eslint --max-warnings 0 --ext .ts,.tsx **/src/*",
+    "lint": "eslint --max-warnings 0 .",
```

Comment:
Do we similarly have to change the `lint-staged` command? Are you seeing the pre-commit runnings for yourself locally?

Reply:
Yes, let me update that. Weirdly the precommit commands ran locally but didn't catch this? I'll see if I can debug why that happened too.

Reply:
🤦  yea adding a TypeScript file repro'd this error in my new clone. I'm still not sure why my original clone isn't running precommit hooks, but I can just switch over to the new one. I'll get this fixed, thanks!

Reply:
Ok this is all fixed, confirmed working! I just removed the second eslint config only being used for precommits. I'm not sure why we'd want to run a different configuration locally and then on CI, seems like an easy way to get tripped up.